OWNERS

```diff
@@ -1,17 +1,19 @@
 reviewers:
-- skeeey
-- zhiweiyin318
+- Jaanki
 - qiujian16
+- skeeey
 - skitt
+- tpantelis
+- zhiweiyin318
 - aswinsuryan
 - dfarrell07
-- Jaanki
 - sridhargaddam
-- tpantelis
 - vthapar
 
 approvers:
-- zhiweiyin318
-- skeeey
+- Jaanki
```

Comment:
I thought we had said we would add approvers according to the admins/owners in the submariner repos, ie skitt, me and mangelajo (tho he's not contributing in this repo right right now).